build.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+##############################################################################
+# Setup script for CEPCSW:
+# - build the cepcsw
+#
+# Usage:
+# $ bash setup.sh
+# or:
+# $ 
```

Comment:
source setup.sh ? I would suggest we just specify one way to set it up

Reply:
Fixed. It's a typo.